docs/configuration/install.md

```diff
@@ -366,15 +366,30 @@ FromLineOverride=YES
 <a name="install-other-depedencies"></a>
 ## Install other Depedencies
 
-On Debian/Ubuntu execute: 
+On Debian/Ubuntu 14.04 execute:
 ```
 apt-get install zip unzip php5-gd php5-mysql php-xsl imagemagick openjdk-7-jdk phantomjs
 ```
 
+On Ubuntu 16.04 execute:
+```
+apt-get install zip unzip php7.0-gd php7.0-mysql php7.0-xsl php7.0-zip imagemagick openjdk-8-jdk phantomjs
+```
+
 On RHEL/CentOS execute: 
 ```
 yum install zip unzip php-gd libxslt ImageMagick java-1.7.0-openjdk phantomjs
 ```
+
+Depending on your use case, you MAY want to install further dependencies (exact package names vary by distribution):
+* php-curl
+* php-xmlrpc
+* php-soap
+* php-ldap
+* php-mbstring
+* ffmpeg
```

Comment:
in 14.04 ffmpeg is called libav-tools, but as you say in the header: exact package names vary by distribution. But maybe it's worth to point that out here

Reply:
Hi! Is the installation of  libapache2-mod-php7.0 not needed?

Reply:
@rrbn, only if you run apache with mod_php. I tried not to make assumptions on the kind of setup here, there are other possible setups too (e.g. php-fpm).

The same is true for the choice of database, but since mysql/mariadb are the only ones officially supported, I think it is safe to assume this kind of setup. As far as php dependencies go, it does not matter if mysql or mariadb is used, since both are supported by the php-mysql binding.